File: docs/snippets/csharp/metadata/PageTrackingBasic.cs
using Kreuzberg;

var config = new ExtractionConfig
{
    Pages = new PageConfig
    {
        ExtractPages = true
    }
};

var result = Kreuzberg.ExtractFileSync("document.pdf", config);

if (result.Pages != null)
{
    foreach (var page in result.Pages)
    {
        Console.WriteLine($"Page {page.PageNumber}:");
        Console.WriteLine($"  Content: {page.Content.Length} chars");
        Console.WriteLine($"  Tables: {page.Tables.Count}");
        Console.WriteLine($"  Images: {page.Images.Count}");
    }
}

File: docs/snippets/php/metadata/page_tracking_basic.php
```php
<?php

declare(strict_types=1);

/**
 * Basic Page Tracking
 *
 * Extract individual pages with their content, tables, and images
 * using page extraction configuration.
 */

require_once __DIR__ . '/vendor/autoload.php';

use Kreuzberg\Kreuzberg;
use Kreuzberg\Config\ExtractionConfig;
use Kreuzberg\Config\PageConfig;

// Configure extraction to include individual pages
$config = new ExtractionConfig(
    pages: new PageConfig(
        extractPages: true
    )
);

$kreuzberg = new Kreuzberg($config);
$result = $kreuzberg->extractFile('document.pdf');

// Process individual pages
if (!empty($result->pages)) {
    foreach ($result->pages as $page) {
        echo "Page {$page->pageNumber}:\n";
        echo "  Content: " . strlen($page->content) . " chars\n";
        echo "  Tables: " . count($page->tables) . "\n";
        echo "  Images: " . count($page->images) . "\n\n";
    }
}
```
